mathics/builtin/system.py

```diff
@@ -29,6 +30,67 @@
     have_psutil = True
 
 
+class MaxLengthIntStringConversion(Predefined):
+    """
+    <url>:Python 3.11:https://docs.python.org/3.11/library/stdtypes.html#int-max-str-digits</url>
+    <dl>
+      <dt>'$MaxLengthIntStringConversion'
+      <dd>A system constant that fixes the largest size of the String resulting from converting
```

Comment:
String -> 'String' here and below.   Integer ->'Integer' 
"converting" -> "converting \"  so there is no hard line break. 

Yes, our docstring formatting is weird. I hope to live to see the day when we can ditch it.